src/components/Modals/NicknameModal/NicknameModal.tsx

```diff
@@ -32,7 +32,7 @@ const NicknameModal = ({ isModalOpen, onRequestClose }: TNicknameModalProps) =>
 
     const history = useHistory();
     const { error: createError, isError, isSuccess, mutate, reset } = api.advertiser.useCreate();
-    const { setHasCreatedAdvertiser } = useAdvertiserInfoState();
+    const { setHasCreatedAdvertiser } = useStore();
```

Comment:
depending on what you values you need from the store, you don't always need to use useShallow, since i'm simply setting something here, in a small component